jnigen/test/README.md

```diff
@@ -0,0 +1,11 @@
+Some notes about tests in this directory:
+
+* `jackson_core_test` is an end-to-end test which generates bindings for `jackson_core` library using the whole jnigen pipeline and compares the generated bindings with expected bindings. `simple_package_test` is similar but instead of using a real Java library from maven, uses a stub java class.
```

Comment:
These could be markdown links so that they are clickable when browsing on Github.